README.md

```diff
@@ -1,61 +1,91 @@
-WordPress for iOS
+# WordPress for iOS #
 
 [![BuddyBuild](https://dashboard.buddybuild.com/api/statusImage?appID=57a120bbe0f5520100e11c19&branch=develop&build=latest)](https://dashboard.buddybuild.com/apps/57a120bbe0f5520100e11c19/build/latest)
 
-## Resources
+## Build Instructions
 
-### Developer blog & Handbook
+### Download Xcode
 
-Blog: http://make.wordpress.org/mobile
+At the moment *WordPress for iOS requires Swift 2.2 and Xcode 7.3 or newer. Previous versions of Xcode can be [downloaded from Apple](https://developer.apple.com/downloads/index.action).*
 
-Handbook: http://make.wordpress.org/mobile/handbook/
+### Third party tools
 
-### Style guide
+We use a few tools to help with development. To install or update the required dependencies, run the follow command on the command line:
 
-https://github.com/wordpress-mobile/WordPress-iOS/wiki/WordPress-for-iOS-Style-Guide
+`rake dependencies`
 
-### To report an issue
+#### CocoaPods
 
-https://github.com/wordpress-mobile/WordPress-iOS/issues
+WordPress for iOS uses [CocoaPods](http://cocoapods.org/) to manage third party libraries.  
+Trying to build the project by itself (WordPress.xcproj) after launching will result in an error, as the resources managed by CocoaPods are not included. To install and configure the third party libraries just run the following in the command line:
 
-### Source Code
+`pod install`
 
-GitHub: https://github.com/wordpress-mobile/WordPress-iOS/
+#### SwiftLint
 
-### How to Contribute
+We use [SwiftLint](https://github.com/realm/SwiftLint) to enforce a common style for Swift code. Xcode will show a warning if you don't have it installed. The app should build and work without it, but if you plan to write code, you are encouraged to install it. No commit should have lint warnings or errors.
 
-http://make.wordpress.org/mobile/handbook/pathways/ios/how-to-contribute/
+SwiftLint runs automatically when you build the project, but you can run it manually from the command line with `rake lint`
```

Comment:
I'd put the command on a new line like you did in the third-party tools section.